Mailer/app/api/Validation/schemas/user-preferences.schema.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "User preferences resource",
+  "type": "object",
+  "required": [
+    "email",
+    "user_id"
+  ],
+  "properties": {
+    "user_id": {
+      "type": "integer"
+    },
+    "email": {
+      "type": "string",
+      "format": "email"
+    },
+    "subscribed": {
+      "type": "integer",
+      "enum": [0,1]
```

Comment:
Why not boolean?

Reply:
It is not represented by boolean in DB, so I used number but I can change it.

Reply:
I am used to boolean type from Pgsql and I forgot that there is no such type in Mysql.

Reply:
Yeah, for some reason they don't have it :/.